clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -111,6 +113,7 @@ export default async function handler(
   // Load the configured consent options (data uses, defaults, etc.) from environment
   const environment = await loadPrivacyCenterEnvironment();
   const serverSettings = await loadServerSettings();
+  const { version } = npmPackage;
```

Comment:
To suggest a bit of scope creep - this is technically the `privacy-center` version (not `fides-js`). We could also expose the `privacy-center` version through a simple `/health` API route as well, which would be helpful.

Reply:
ok, got this working at http://localhost:3001/api/health